packages/cicero-cli/package.json

```diff
@@ -49,8 +49,8 @@
     "@accordproject/cicero-core": "0.10.2",
     "@accordproject/cicero-engine": "0.10.2",
     "@accordproject/cicero-tools": "0.10.2",
-    "@accordproject/ergo-compiler": "0.6.4-20190313182634",
-    "composer-concerto": "0.60.0",
+    "@accordproject/ergo-compiler": "0.6.4-20190319173619",
```

Comment:
Should we release Ergo Compiler first and ref a stable release?

Reply:
Yes. That was just a way to test with the latest dev version. I'm still unwinding this branch.